flask-lesson_1/homework/fruits/routes.py

```diff
@@ -0,0 +1,37 @@
+from flask import Blueprint, render_template, request, redirect
+
+fruits = Blueprint('fruits', __name__)
+
+fruits_list = ['Apple', 'Guava', 'Orange', 'Banana']
+
+
+# @fruits.route('/fruits')
+# @fruits.route('/fruits/<string:value>', methods=['GET', 'POST', 'DELETE'])
+# def fruit_func(value=None):
+#     if request.method == "POST":
+#         fruits_list.append(value)
+#         return "Fruit was added successfully"
+#     elif request.method == "DELETE":
+#         fruits_list.remove(value)
+#         return "Fruit was deleted successfully"
+#     else:
+#         return render_template("fruits.html", title='Fruits Page!', fruit_list=fruits_list)
+
+
+
+@fruits.route("/fruits")
+@fruits.route('/fruits/<string:value>', methods=['GET'])
```

Comment:
if we type `/fruits` we will see all the fruits, but if we type `/fruits/some/` we have to see info that belongs to only `some` fruit

Reply:
done